README.rst

```diff
@@ -136,11 +136,25 @@ The official documentation is hosted on ReadTheDocs at http://larray.readthedocs
 Get in touch
 ============
 
-- Report bugs, suggest features or view the source code `on GitHub`_.
-- For questions, ideas or general discussion, use the `mailing list`_.
-
-.. _on GitHub: http://github.com/liam2/larray
-.. _mailing list: https://groups.google.com/forum/#!forum/larray
+- To be informed of each new release, please subscribe to the announce `mailing list`_.
+- For questions, ideas or general discussion, please use the `Google Users Group`_.
+- To report bugs, suggest features or view the source code, please go to our `GitHub website`_.
+
+.. _mailing list: https://groups.google.com/d/forum/larray-announce
+.. _Google Users Group: https://groups.google.com/d/forum/larray-users
+.. _GitHub website: http://github.com/liam2/larray
+
+.. raw:: html
+
+   <p>
+   You can subscribe to the announce mailing list by entering your email address here
+   (if you are connected to your Google account but you want to subscribe using another address,
+    please log out first):
+   </p>
+   <form action="https://groups.google.com/group/larray-announce/boxsubscribe">
```

Comment:
the form does not seem to render on github (but does render on RTD) but the `<p>` leading it does, making it look weird on Github, so I suggest you move the whole ".. raw" thing to doc/source/index.rst. The result should be the same on RTD.